spynnaker/pyNN/models/neuron/synaptic_manager.py

```diff
@@ -1220,7 +1227,11 @@ def write_data_spec(
                                        m_edge.pre_vertex.vertex_slice] = \
                     routing_info.get_routing_info_for_edge(m_edge)
 
-        post_slices = application_vertex.vertex_slices
+        post_slices, is_exact = (
```

Comment:
Would it not be better to use get_pre_vertices even if that does require getting the outgoing_edge_partition passed into here.